src/emhass/forecast.py

```diff
@@ -151,30 +151,30 @@ def __init__(self, retrieve_hass_conf: dict, optim_conf: dict, plant_conf: dict,
                     self.forecast_dates = self.forecast_dates[0:self.params['passed_data']['prediction_horizon']]
         
         
-    def get_weather_forecast(self, method: Optional[str] = 'scrapper',
+    def get_weather_forecast(self, method: Optional[str] = 'scraper',
                              csv_path: Optional[str] = "/data/data_weather_forecast.csv") -> pd.DataFrame:
         r"""
         Get and generate weather forecast data.
         
-        :param method: The desired method, options are 'scrapper', 'csv', 'list' and 'solcast'. \
-            defaults to 'scrapper'
+        :param method: The desired method, options are 'scraper', 'csv', 'list' and 'solcast'. \
+            defaults to 'scraper'
         :type method: str, optional
         :return: The DataFrame containing the forecasted data
         :rtype: pd.DataFrame
         
         """
         self.logger.info("Retrieving weather forecast data using method = "+method)
         self.weather_forecast_method = method # Saving this attribute for later use to identify csv method usage
-        if method == 'scrapper':
-            freq_scrap = pd.to_timedelta(60, "minutes") # The scrapping time step is 60min
+        if method in ('scraper', 'scraper'):  # typo compatibility
```

Comment:
I guess this should be backwards compatible.

```suggestion
        if method in ('scraper', 'scrapper'):  # typo compatibility
```